corehq/reports.py

```diff
@@ -347,6 +348,12 @@ def get_report_builder_count(domain):
     )),
 )
 
+ENTERPRISE_INTERFACES = (
```

Comment:
Same issue with naming here. I see that there is a precedent for 'interfaces' here, i.e. `EDIT_DATA_INTERFACES` and `SMS_ADMIN_INTERFACES`, but it also has `ADMIN_REPORTS` and `DOMAIN_REPORTS`. What makes something a report vs an interface?

Reply:
an interface generally has more actions associated with it (like a "create" action). I agree in this case the naming could be report as the original SMSBillables interface is also really a report more than an interface. This is a wider issue with the reporting framework though. The naming is consistent with how things are set up on the billing side

Reply:
I'm keeping this as it is what is done in SMS billables and I would like to avoid confusion by not changing things too much.